### DISCUSSION TITLE
installation

### DISCUSSION THREAD

**Discussion by TouKeerAhmed** [softwarecrash/Solar2MQTT-ESP8266#100] [General]
Download the [latest Firmware](https://github.com/softwarecrash/Solar2MQTT/releases/latest)
flash the bin file to an ESP8266 (recommended Wemos D1 Mini) with [Tasmotizer](https://github.com/tasmota/tasmotizer/releases)
connect the ESP like the [wiring diagram](https://github.com/softwarecrash/Solar2MQTT/wiki/Wiring-Diagram)
search for the wifi ap "Solar2MQTT-AP" and connect to it
surf to 192.168.4.1 and set up your wifi and optional MQTT

after that how to access the interface ? there is no ip address  in router too. although it is connected to wifi with mac.

**Comment by softwarecrash** [comment]
Sound Something is wrong, reset and connect it again
